VideoAPI/VideoApi.DAL/Queries/GetClosedConferencesQuery.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+using VideoApi.Domain.Enums;
+
+namespace VideoApi.DAL.Queries
+{
+    public class GetClosedConferencesQuery : IQuery
+    {
+    }
+
+    public class GetClosedConferencesHandler : IQueryHandler<GetClosedConferencesQuery, List<Conference>>
```

Comment:
Again here you are getting closed conferences with instant messages. Please rename to be more reflective (i.e. GetClosedConferencesWithInstantMessages)

Reply:
renamed the classes to GetClosedConferencesWithInstantMessagesQuery and GetClosedConferencesWithInstantMessagesHandler